test/unit/connectors/ApiNotificationQueueConnectorSpec.scala

```diff
@@ -172,8 +170,9 @@ class ApiNotificationQueueConnectorSpec extends UnitSpec with ScalaFutures with
 
       await(connector.delete(notification))
 
-
-      wverify(deleteRequestedFor(url))
+      eventually {
```

Comment:
Consider using our WireMockRunner trait? And use port/host from `customs-api-common`?